### PR TITLE
application layout using bootstrap

### PR DESCRIPTION
**Pull request by aleemchishti**
Changed layout of the application mostly and fixed some migration errors

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -5,6 +5,6 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   
   enum role: [:sender, :traveler]
-  has_one :journey
+  has_many :journey
```
Comment (momnanaseer) [inline app/models/user.rb:8]: it should be `has_many :journeys` not `has_many :journey`

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -64,7 +64,7 @@
       </div>
     </nav>
     <div
-      class="bg-image"style="background-image: url('https://img.freepik.com/premium-photo/smooth-empty-grey-studio-well-use-as-background_1258-3149.jpg?w=2000');height: 120vh;"> 
+      class="bg-image"style="background-image: url('https://www.purolatorinternational.com/content/uploads/2021/12/iStock-1124533318-1.jpg');height: 100vh;">
```
Comment (momnanaseer) [inline app/views/layouts/application.html.erb:67]: shift this styling into css file.

File: app/views/orders/_form.html.erb [hunk 3]
```diff
@@ -4,7 +4,7 @@
   <br>
   <div class="form-group">
       <%= form.label :travelers, class:'form-label mb-2 mt-3' %><br />
-      <%= form.select :travelers,User.all.traveler.pluck(:first_name),{prompt:'Select'},{class: "form-control"}%>
+      <%= form.select :travelers,User.traveler.pluck(:first_name),{prompt:'Select'},{class: "form-control"}%>
```
Comment (momnanaseer) [inline app/views/orders/_form.html.erb:7]: use instance variable here, query in view is not good approach.
add this line in controller and use @users variable in view
`@users  = User.all.traveler.pluck(:first_name)`

File: app/views/orders/_form.html.erb [hunk 4]
```diff
@@ -4,7 +4,7 @@
...
   <div class="form-group">
       <%= form.label :from, class:'form-label mb-2 mt-3' %><br />
       <%= form.select :from,Journey.all.pluck(:from),{prompt:'Select'},{class: "form-control"}%>
```
Comment (momnanaseer) [inline app/views/orders/_form.html.erb:10]: same instance variable technique for this line too.

File: app/views/traveler/journeys/show.html.erb [hunk 5]
```diff
@@ -2,21 +2,12 @@
 <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
 <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
```
Comment (momnanaseer) [inline app/views/traveler/journeys/show.html.erb:3]: shift these lines in layout file.

File: app/models/user.rb (hunk 1 above)
Comment (aleemchishti) [reply app/models/user.rb:8]: typing mistake.fixed

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (aleemchishti) [reply app/views/layouts/application.html.erb:67]: rails 7 is problematic. Working on it. Couldn't work with custom css files before

File: app/views/orders/_form.html.erb (hunk 3 above)
Comment (aleemchishti) [reply app/views/orders/_form.html.erb:7]: i have modified this file already. Will push the code after changes completed

File: app/views/orders/_form.html.erb (hunk 4 above)
Comment (aleemchishti) [reply app/views/orders/_form.html.erb:10]: okay

File: app/views/traveler/journeys/show.html.erb (hunk 5 above)
Comment (aleemchishti) [reply app/views/traveler/journeys/show.html.erb:3]: okay

File: app/views/orders/_form.html.erb (hunk 4 above)
Comment (aleemchishti) [reply app/views/orders/_form.html.erb:10]: fixed

File: app/views/traveler/journeys/show.html.erb (hunk 5 above)
Comment (aleemchishti) [reply app/views/traveler/journeys/show.html.erb:3]: done

**Pull request merged**